tools/appraisal_helper.rb

```diff
@@ -0,0 +1,36 @@
+def ruby_version(version, java_required = false, &block)
```

Comment:
What's this file used for?

Reply:
Oh I see, the huge diff in Appraisals was collapsed above.

Reply:
Instead of adding yet another big block of hard to maintain code to Appraisals, I've quickly wrote those helpers to reduce the duplication considerable as this PR is `+359 −562`.